packaging_ignore.yml

```diff
@@ -13,6 +13,8 @@ base:
   - ".* unused substitution variable \\$\\{misc:Depends\\}.*"
   - ".*: substitution variable \\$\\{postgresql:Depends\\} unused, but is defined.*"
   - ".* unused substitution variable \\$\\{postgresql:Depends\\}.*"
+  - ".* unused substitution variable \\$\\{shlib:Depends\\}.*"
+  - "WARNING generated by debuild.*"
```

Comment:
I think we should continue to see those warnings generated by debuild as some of them may be helpful.

For example, certain versions of debuild may complain that we have empty files in a package ( See `debian/docs` for an example ) and this is something that is better fixed rather than ignored IMO.

Reply:
bump

Reply:
Opened an issue to handle this message without adding it into ignore file
https://github.com/citusdata/tools/issues/232